### ISSUE TITLE
Fix an unexpected error with `satisfies` keyword in TSX files

### ISSUE THREAD

**Issue by tnyo43** [markuplint/markuplint#1440]
- Markuplint version: `4.0.1`
- Parser lang: tsx
- Node.js version: `20.11.0`
- OS: macOS

## Describe the bug

When I use `as const satisfies Record<X, Y>` in tsx files, it throws an unexpected error.

## Code Example

```tsx
type Key = "name" | "address";

const x = {
  name: "Alice",
  address: "UK",
} as const satisfies Record<Key, string>;

const C = () => {
  return <div />;
};

```

## Steps To Reproduce

1. clone https://github.com/tnyo43/markuplint-satisfies-error
2. run `pnpm install & pnpm run lint:markup` and you will see an error as like follows:

## The current behavior

I encounter an error when I run `pnpm run lint:markup`.

```
<markuplint> error: Error: Unsupported node (parse-error) /path/to/markuplint-satisfies-error/src/C.tsx:0:0
  0: type Key = "name" | "address";
  1: type•Key•=•"name"•|•"address";
  2: const x = {
  3: ••name: "Alice",
  4: ••address: "UK",
  5: } as const satisfies Record<Key, string>;
  6: }•as•const•satisfies•Record<Key,•string>;
  7: const C = () => {
  8: ••return <div />;
  9: };return•<div•/>;
  10: };
  11: 
 ELIFECYCLE  Command failed with exit code 1.
```

## The expected behavior

no error
